api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -497,6 +501,12 @@ public Profile updateAccountProperties(AccountPropertyUpdate request) {
     request
         .getAccessBypassRequests()
         .forEach(bypass -> userService.updateBypassTime(dbUser.getUserId(), bypass));
+    request
```

Comment:
please add the "dual write and deprecate" comment here like the other calls to updateBypassTime()